### PR TITLE
Various validation error fixes

### PR DESCRIPTION
**Pull request by 36degrees**
See individual commits for details.

### CODE REVIEW

File: spec/features/topic_spec.rb [hunk 1]
```diff
@@ -114,6 +114,7 @@
 
     within '.full-error-list' do
       expect(page).to have_content "Path must be present and start with '/service-manual/'"
+      expect(page).to have_content "Title can't be blank"
```
Comment (thehenster) [inline spec/features/topic_spec.rb]: It'd be nice to test this in the model spec. There _should_ be similar assertions on the existing validations in there..

File: app/assets/stylesheets/topics.scss [hunk 2]
```diff
@@ -1,4 +1,4 @@
-.topics {
+.topic-section-list {
```
Comment (thehenster) [inline app/assets/stylesheets/topics.scss]: :+1:

How about renaming the file to topic-section-list.scss? I like it when the block name is reflected in the filename. We follow that pattern in service-manual-frontend. Not that important so do what feels right.

File: app/views/topics/_form.html.erb [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<div class="topics">
+<div class="row">
```
Comment (thehenster) [inline app/views/topics/_form.html.erb]: Good removal of that topics class too :+1:

**Comment by thehenster** [comment]
LGTM

File: app/assets/stylesheets/topics.scss (hunk 2 above)
Comment (36degrees) [inline app/assets/stylesheets/topics.scss]: This hasn't been marked as outdated, but has been done.

**Pull request merged**